vsintegration/Directory.Build.targets

```diff
@@ -2,27 +2,20 @@
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
 
-  <ItemGroup Condition="'$(NoMsbuild)' != 'true'">
-    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" PrivateAssets="all" ExcludeAssets="contentFiles;analyzers;native" />
+  <ItemGroup>
```

Comment:
This is probably the most controversial change. I noticed that the `NoMsbuild` condition didn't seem to ever be set, and so by removing it we can consolidate and sort the list of PackageReferences for easier comparison.